mhs/mhs/common/state/persistence_adaptor.py

```diff
@@ -0,0 +1,36 @@
+"""
+This module defines the state adaptor interface, used to allow support for multiple state database implementations.
+"""
+import typing
+
+
+class PersistenceAdaptor:
```

Comment:
Should we make this subclass from `abc.ABC` just to show it's an interface

Reply:
done